### DISCUSSION TITLE
How to use Comparison Charts

### DISCUSSION THREAD

**Discussion by bequrios** [visualize-admin/visualization-tool#1385] [General]
It seems, there is the very handy feature of comparison charts in visualize 
![grafik](https://github.com/visualize-admin/visualization-tool/assets/22146779/af23c382-cd11-4985-b8e7-03797986226d)

How can i create such comparison charts and what are the prerequisites concerning the data to allow such charts? I could not manage to create such graphs so far...

**Comment by bprusinowski** [comment]
Hi @bequrios, thanks for starting a discussion!

In order to be able to create a comparison chart, a given dataset needs to have:
- temporal dimension,
- at least two measure dimensions.

If there are at least two measures with different units (e.g. CHF and °C), dual-axis line and column-line comparison charts will be enabled; in case there are at least two measures with the same unit, multi-line comparison chart will be enabled.

One dataset to check out the feature would be [Einmalvergütung für Photovoltaikanlagen](https://int.visualize.admin.ch/en/create/new?cube=https://energy.ld.admin.ch/sfoe/bfe_ogd84_einmalverguetung_fuer_photovoltaikanlagen/9&dataSource=Prod).

Let me know if it makes sense, or if the feature doesn't work for some datasets that theoretically should support it.
